### DISCUSSION TITLE
Use local DEM

### DISCUSSION THREAD

**Discussion by OAStubby** [devemux86/cruiser#172] [Q&A] [answered]
It may be useful to be able to use DTM/DEM, such as these, [https://sonny.4lima.de/](https://sonny.4lima.de/), loaded from a local folder for hillshade generation.

**Comment by devemux86** [comment, marked as answer]
@OAStubby thanks for the idea!

This may be possible for the [Mapsforge](https://github.com/mapsforge/mapsforge) map engine of Cruiser.

[VTM (OpenGL)](https://github.com/mapsforge/vtm) map engine renders the map in real time and something like that is not currently supported.

**Comment by ppete2** [comment]
I support this idea! It would be great if Cruiser can use local .hgt elevation files stored on the device to display Hillshading. First these could be also used completely offline, second especially Sonny's DTM files from https://sonny.4lima.de are more accurate than online ESRI-shading.

Maybe in future releases of Cruiser these .hgt files could also be used for displaying the elevation of a spot or calculate elevation sums of routes.
